apps/server/src/server.ts

```diff
@@ -35,6 +52,6 @@ app.use('/health', healthRouter);
 
 app.use(errorHandler);
 // running the server
-app.listen(serverConfig.port, () => {
-	console.log(`Starting Express server on http://localhost:${serverConfig.port}`);
+app.listen(port, () => {
+	console.log(`Starting Express server on http://localhost:${port}`);
```

Comment:
the server doesn't know its host, and "localhost" may be misleading.
```suggestion
	console.log(`Starting ExpressJS server on port ${port}`);
```